packages/astro/src/core/config.ts

```diff
@@ -23,6 +23,38 @@ interface PostCSSConfigResult {
 	plugins: Postcss.Plugin[];
 }
 
+export const astroConfigDefaults: AstroUserConfig & any = {
```

Comment:
- This probably doesn't need to be exported, does it?
- We tend to use `CONSTANT_CASE` for things like this, would you mind updating?
- Curious why the `& any` is used here. Maybe dropping this type and using `as const` at the end would be better?

Reply:
- I agree, CONSTANT_CASE better in this case.
- Exported because: ex. if somebody want to test the defaults. but I will remove it and then if someone needs it then they will export.